JMS/com/ibm/mq/samples/jms/SampleEnvSetter.java

```diff
@@ -61,13 +61,19 @@ public SampleEnvSetter() {
             if (mqEnvSettings != null) {
               logger.info("JSON Data Found");
               mqEndPoints = (JSONArray) mqEnvSettings.getJSONArray("MQ_ENDPOINTS");
+              if (mqEnvSettings.has("JWT_ISSUER")) {
```

Comment:
I think this is ok, but the clause becomes `if (mqEnvSettings != null) && mqEnvSettings.has("JWT_ISSUER"))` 

It makes sense to put this outside the `if (mqEnvSettings != null)`